rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -90,6 +90,7 @@ def _insert_order(self, participant, order_id, tests, order_time, finalized_test
         participantId=participant.participantId,
         sourceSiteId=1,
         finalizedSiteId=1,
+        collectedSiteId=1,
```

Comment:
You may want to clear out finalizedSiteId here, or set it to something else, to prove collected site ID is being used now.

We should add a check here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/biobank_order_dao.py#L246

To ensure that all biobank orders created henceforth have collected site info. (Which appears to be the case now, but wasn't the case back in June of last year.)

Reply:
set to 2